registries.conf

```diff
@@ -0,0 +1,22 @@
+# This is a system-wide configuration file used to
+# keep track of registries for various container backends.
+# It adheres to TOML format.
+
+# The default location for this configuration file is /etc/containers/registries.conf.
+
+# The only valid categories are: 'secure', 'insecure' and 'block'.
+
```

Comment:
Perhaps add something about multiples?:  "If you need to list multiple registries in a category, each registry should be quoted and separated with a comma.  As an example ["registry.mydomain.com", "registry.access.redhat.com"].  The registries will be searched in order until the requested image is located.